### PR TITLE
refactor by renaming functions

### PR DESCRIPTION
**Pull request by sjswerdloff**
prefix with _ to make all functions private other than anonymize()
	change to names that more clearly indicate purpose
added docstring to more functions
eliminated some unused imports
eliminated use of LOAD_DCM function whose purpose had essentially been eliminated in some previous change from re-reading DICOM files to indexing in to dictionary of patient's DICOM objects.
eliminated indexing in to dictionary of patient's DICOM objects in loops that iterated on the dictionary of patient DICOM objects
by iterating on the items (getting the key and value simultaneously)
reworded text in some of the print statements for clarity

### CODE REVIEW

File: src/Model/Anon.py [hunk 1]
```diff
@@ -405,50 +390,55 @@ def Print_identifiers(ds_rtss):
...
-    )
-
-    # concatinating the folder path and the filename
-    Full_dicom_filepath = Dicom_folder_path + "/" + Dicom_filename
-    print("In Load_DCM function: FULL PATH of dicom file is:----", Full_dicom_filepath)
-    # ds_rtss = pydicom.dcmread(Full_dicom_filepath)
-    ds_rtss = new_dict_dataset[key]
-    print("In Load DCM function:", Dicom_filename, "loaded in ds_rtss")
-    return ds_rtss
+def _is_directory(file_path):
```
Comment (sjswerdloff) [inline src/Model/Anon.py:393]: renamed from Check_if_folder

File: src/Model/Anon.py [hunk 2]
```diff
@@ -405,50 +390,55 @@ def Print_identifiers(ds_rtss):
...
 
 
-def Check_if_folder(file_path):
-    # store the boolean value after checking the type of file
-    file_type = os.path.isdir(file_path)
-    if file_type == True:  # if the file is subdiirectory return true
-        return True
-    else:
-        return False  # if not a subdirectory return False
+def _check_file_hashed(file_name, new_dict_dataset, key, matching_text):
```
Comment (sjswerdloff) [inline src/Model/Anon.py:400]: also renamed positional argument

**Comment by sjswerdloff** [comment]
This is probably close to the "too many changes at once" end of the scale for a PR.

**Comment by kylelw23** [comment]
This is a very thoughtful work. The "_" prefix has shown a clear concept of what can be imported and what can not be.

**Pull request merged**